vncviewer/GestureHandler.cxx

```diff
@@ -25,7 +26,17 @@
 
 static rfb::LogWriter vlog("GestureHandler");
 
-static const int GH_INITSTATE = 255;
+static const unsigned char GH_NOGESTURE = 0;
+static const unsigned char GH_ONETAP    = 1;
+static const unsigned char GH_TWOTAP    = 2;
+static const unsigned char GH_THREETAP  = 4;
+static const unsigned char GH_DRAG      = 8;
+static const unsigned char GH_THREEDRAG = 16;
+static const unsigned char GH_LONGPRESS = 32;
+static const unsigned char GH_PAN       = 64;
+static const unsigned char GH_PINCH     = 128;
```

Comment:
Why not use the ones from GestureEvent?

Reply:
@CendioOssman explained that there is no motivation for the GestureEvents to be a bitmask. GestureEvent definitions shouldn't be required to have knowledge about the state machine which uses these constants in GestureHandler.